db/schema.rb

```diff
@@ -91,7 +91,6 @@
     t.string "unlock_token"
     t.datetime "locked_at"
     t.string "true_layer_secure_data_id"
-    t.datetime "remember_created_at"
```

Comment:
Should this be removed?

Reply:
Not sure why the schema changed as I didn't touch it :/ I'm reading that rails updates the schema.rb file using the database as a source, so can make unexpected changes if developers use different environments?